main.py

```diff
@@ -603,6 +603,15 @@ def handle_flows_install_delete(self, event):
             return
 
         force = bool(event.content.get("force", False))
+        if not flow_dict["flows"]:
+            log.error(f"Error, empty list of flows recieved. {flow_dict}")
+            return
+        else:
+            log.info(
+                f"Received event with {len(flow_dict['flows'])} flows"
```

Comment:
Functionality-wise it accomplishes it purposes, but for easy to follow readability of the messages can you include the len of the total flows on the first log.info execution of `flows_to_log_info`. That way it'll be easier to follow since if you're interested in knowing the total you just have to find the first message of the slice. 

Currently you need to follow more than one, under normal circumstances they'll be adjacent to each other but it's more things to follow and correlate, making it harder to debug especially on elastic search/kibana:

```
2024-08-15 09:55:07,377 - INFO [kytos.napps.kytos/flow_manager] (AnyIO worker thread) Received request with 1 flows_dict with force=False.
2024-08-15 09:55:07,377 - INFO [kytos.napps.kytos/flow_manager] (AnyIO worker thread) Send FlowMod from request dpid: 00:00:00:00:00:00:00:03, command: add, force: False,  flows[0, 1]: [
{'match': {'in_port': 2, 'dl_vlan': 3}, 'cookie': 12282679257177192524, 'actions': [{'action_type': 'pop_vlan'}, {'action_type': 'output', 'port': 1}], 'owner': 'mef_eline', 'table_group
': 'evpl', 'table_id': 0, 'priority': 20000}]
```

Reply:
Added length to first log for partitions.